app/components/AccountProfile.tsx

```diff
@@ -0,0 +1,76 @@
+import { t } from "i18next";
+import Avatar from "./Avatar";
+import { Text } from "@radix-ui/themes";
+import styles from "./accountProfile.module.css";
+import { FollowButton } from "./FollowButton";
+
+type AccountProfileProps = {
+  /** account following status.
+   * - notFollowing: not following this account
+   * - following: following this account
+   * - self: account is self
+   * - blocking: blocking this account
+   */
+  accountFollowStatus: "notFollowing" | "following" | "self" | "blocking";
+  /** is followed by this account */
+  isFollowed: boolean;
+  avatarUrl: string;
+  headerUrl: string;
+  nickname: string;
+  accountName: string;
+  bio: string;
+  followersCount: number;
+  followingCount: number;
+};
+
+export const AccountProfile = ({
+  accountFollowStatus,
+  isFollowed,
+  avatarUrl,
+  headerUrl,
+  nickname,
+  accountName,
+  bio,
+  followersCount,
+  followingCount
+}: AccountProfileProps) => {
+  return (
+    <>
+      <img
+        className={styles.accountProfileHeader}
+        src={headerUrl}
+        alt={`${nickname} header image`}
+      />
+
+      <div className={styles.accountProfileInfo}>
+        <Avatar
+          {...{ name: accountName, nickname: nickname, iconUrl: avatarUrl }}
+        />
+        <FollowButton {...{ accountFollowStatus, isFollowed }} />
+      </div>
+
+      <div className={styles.accountProfileStatus}>
+        <Text size={"3"}>{bio}</Text>
```

Comment:
```suggestion
        <Text size="3">{bio}</Text>
```
常にプロパティに `{}` を用いるというルールではないと思うので, 文字列リテラルの指定としてこれは冗長ではないでしょうか